### PR TITLE
[BCHDCC-42]: Label in Name & [BCHDCC-43]: Labels and Instructions

### PR DESCRIPTION
**Pull request by emmabjj**
## Description

WCAG 2.5.3: As a user that requires a screen reader, I want to rely on visible labels as a means to interact with the components, so I can navigate and select features throughout a web page. 

WCAG 3.3.2: As a user that requires a screen reader, I want to know the labels of the form fields and which form fields are required, so I can complete a form without being stuck.

Clearer labels and instructions were added to User Profile and Forgot Your Password pages. 

## Motivation and Context

[BCHDCC-42](https://app.clickup.com/t/9006094761/BCHDCC-42) - Accessibility Criterion 2.5.3: Label in Name
[BCHDCC-43](https://app.clickup.com/t/9006094761/BCHDCC-43) - Accessibility Criterion 3.3.2: Labels or Instructions

## Screenshots

**Libraries Added**

## Internal Checklist

- [x] My code follows the code style of this project (https://rubystyle.guide/).
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by anna-dorigo** [comment]
IT looks great! just a quick comment: 
Current password instructions and field appears twice in the form
image

File: app/views/devise/passwords/new.html.haml [hunk 1]
```diff
@@ -1,13 +1,15 @@
 - title "Forgot Your Password"
 .authform.devise
-  = content_tag(:span, nil, class: 'fa fa-key fa-3x', style: "display: block")  
-  = form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post, role: 'form' }) do |f|
-    %h2 Forgot your password?
+  = content_tag(:span, nil, class: 'fa fa-key fa-3x', style: "display: block; text-align: center;")  
+  = form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post, role: 'form', class: 'password-reset-form' }) do |f|
+    %h2.forgot-password-title Forgot your password?
+    %h3.instructions Enter your email and we will send you instructions to reset your password.
```
Comment (anna-dorigo) [inline app/views/devise/passwords/new.html.haml:6]: ✨

File: app/views/devise/registrations/edit.html.haml [hunk 2]
```diff
@@ -20,16 +20,22 @@
...
+      You must enter your current password to make changes.
+    .form-group
+      = f.label :current_password
+      = f.password_field :current_password, autocomplete: 'off', class: 'form-control'    
+  %fieldset
     %p
       Leave these fields blank if you don't want to change your password.
     .form-group
-      = f.label :password
+      = f.label :password, "New password"
```
Comment (anna-dorigo) [inline app/views/devise/registrations/edit.html.haml:25]: ✨

**Comment by emmabjj** [comment]
@anna-dorigo - I fixed the double "current password", it wasn't my original change - I think that change was accidentally made when resolving a merge conflict using the github UI. Should be good to go now! :) 👍

**Review by anna-dorigo — APPROVED** [review]
👍 Great Work Emma 🎉

**Pull request merged**